projects/fsastorefrontlib/src/cms-components/my-account/policy/policies-chart/policies-chart.component.html

```diff
@@ -0,0 +1,30 @@
+<div *ngIf="policiesByPaymentFrequency" class="accordion mb-4">
+  <cx-fs-accordion-item
+    [opened]="false"
+    [icon]="'fas fa-chart-pie'"
+    [title]="'policy.policyChartTitle' | cxTranslate"
+  >
+    <div class="row">
+      <label class="col-md-3 pr-3 pr-md-0">
+        <span class="label-content">{{
+          'fscommon.paymentFrequency' | cxTranslate
+        }}</span>
+        <select
```

Comment:
Since ng-select module is available (but unused), maybe use it instead of regular select. That way it may be easier to fix values not changing in select once language is switched.

Reply:
I'll use `ng-select` here and remove `FormsModule` from policy.module